test/lint/lint-circular-dependencies.sh

```diff
@@ -79,9 +72,9 @@ EXPECTED_CIRCULAR_DEPENDENCIES=(
     "core_io -> evo/cbtx -> evo/simplifiedmns -> core_io"
     "evo/simplifiedmns -> llmq/blockprocessor -> net_processing -> evo/simplifiedmns"
     "llmq/dkgsession -> llmq/dkgsessionmgr -> llmq/dkgsessionhandler -> llmq/dkgsession"
-    "logging -> util/system -> random -> logging"
     "logging -> util/system -> sync -> logging"
     "logging -> util/system -> stacktraces -> logging"
+    "logging -> util/system -> util/getuniquepath -> random -> logging"
```

Comment:
Added "logging -> util/system -> util/getuniquepath -> random -> logging" as reported by linter that the new dependency is introduced